beerus-rpc/src/beerus_rpc_server.rs

```diff
@@ -13,13 +16,27 @@ pub struct BeerusRpc {
 trait BeerusApi {
     #[method(name = "hello_world")]
     async fn hello_world(&self) -> Result<String>;
+
+    #[method(name = "stark_nonce")]
+    async fn stark_nonce(&self, contract_address: String) -> Result<String>;
 }
 
 #[async_trait]
 impl BeerusApiServer for BeerusRpc {
     async fn hello_world(&self) -> Result<String> {
         Ok("Hello World!".to_string())
     }
+
+    async fn stark_nonce(&self, contract_address: String) -> Result<String> {
+        let contract_address = FieldElement::from_str(&contract_address).unwrap();
```

Comment:
As we are trying to get std out of the library what are your thoughts on defaulting to 
`FieldElement::from_hex_be` for the time being?

Reply:
Yes, using `FieldElement::from_hex_be` instead of `FieldElement::from_str` could be a good solution if we are certain that the input is in `hexadecimal` format, which would allow us to avoid the dependency on `std`.

If the input could be in a different format, such as `decimal`, then we could consider writing a custom conversion function that does not depend on `std`.

I think that in this case using the `FieldElement::from_hex_be` function would be sufficient.

Reply:
Okay, lets stick to `from_hex_be` for the time being, and if this comes up as a feature request or issue later we can add the conversion

Reply:
@danilowhk thoughts here?

Reply:
I already use the `from_hex_be` method